docs/design/features/globalization-invariant-mode.md

```diff
@@ -104,7 +104,7 @@ Applications can enable the invariant mode by either of the following:
 
     ```xml
     <ItemGroup>
-      <RuntimeHostConfigurationOption Include="System.Globalization.Invariant" Value="true" />
+        <InvariantGlobalization>true</InvariantGlobalization>
```

Comment:
@eerhardt do you know if this is something changed?

Reply:
The only meaningful change was that the ILLinker added a new metadata item “Trim=true” to this item in order to be respected by the linker. So the current MSBuild xml in the docs isn’t enough for the linker to work.

Using the property, which is the proposed change, is the right call. We should be telling users to set the properties.